### PR TITLE
Adding visually if player droped items, selected option and additional attempt to playback a broken replay

### PR DESCRIPTION
**Pull request by lmsv-mx123**
Adding visually if player droped items, selected option and additional attempt to playback a broken replay, notably: "07-24-2018 12.31.52 currently broken reconnect, but i think some badwords testing, thieving, and going to draynor to maybe get disk of returning"

### CODE REVIEW

File: src/Game/ReplayServer.java [hunk 1]
```diff
@@ -321,8 +346,24 @@ public boolean doTick() {
...
             Replay.timestamp = timestamp_input;
+          
+					disconnected = true;
         }
-      }
+			}
+			
+			// TODO: this only will skip 1 keys, unsure if there is a replay where more than 1 isaac key
+			// should be skipped from OG replays
+			if (nextIncomingPacket != null && nextIncomingPacket.opcode == 10000
```
Comment (Hubcapp) [inline src/Game/ReplayServer.java:356]: This is kind of weird b/c it ties the opcode logger functionality (which is expected to only be enabled if the Logging level is set high enough) to replay playback functionality.

File: src/Game/ReplayServer.java [hunk 2]
```diff
@@ -258,15 +278,16 @@ public boolean doTick() {
           }
         }
       } else {
-        if (Settings.LOG_VERBOSITY.get(Settings.currentProfile) >= 5) {
+				if (Settings.LOG_VERBOSITY.get(Settings.currentProfile) >= 3) {
```
Comment (Hubcapp) [inline src/Game/ReplayServer.java:281]: \>= 5 || Settings.parseOpcodes

File: src/Game/ReplayServer.java [hunk 3]
```diff
@@ -177,7 +177,7 @@ public void run() {
           incomingPacketsIndex = 0;
           outgoingPacketsIndex = 0;
 
-          if (Settings.LOG_VERBOSITY.get(Settings.currentProfile) >= 5) {
+					if (Settings.LOG_VERBOSITY.get(Settings.currentProfile) >= 3) {
```
Comment (Hubcapp) [inline src/Game/ReplayServer.java:180]: \>= 5 || Settings.parseOpcodes

File: src/Game/ReplayServer.java [hunk 4]
```diff
@@ -128,7 +128,7 @@ public void run() {
       Logger.Debug("ReplayServer: Replay loaded, waiting for client; length=" + timestamp_end);
 
       // Load replay a second time but using the RSCMinus method
-      if (Settings.LOG_VERBOSITY.get(Settings.currentProfile) >= 5) {
+			if (Settings.LOG_VERBOSITY.get(Settings.currentProfile) >= 3) {
```
Comment (Hubcapp) [inline src/Game/ReplayServer.java:131]: \>= 5 || Settings.parseOpcodes

File: src/Game/ReplayServer.java [hunk 5]
```diff
@@ -231,21 +231,41 @@ public boolean doTick() {
...
                 nextOutgoingPacket.data);
+						if (nextOutgoingPacket.opcode == 246) {
+							int item = -1;
+							try {
+								item = nextOutgoingPacket.data[0] << 8 | nextOutgoingPacket.data[1];
+							} catch (Exception e) {
+							}
+							if (item < 0)
+								continue;
+							Client.displayMessage("Dropping " + Item.item_name[Client.inventory_items[item]], Client.CHAT_NONE);
```
[the remark this answers is not in the document]
Comment (lmsv-mx123) [reply src/Game/ReplayServer.java:249]: That is an authentic message that always has been client sided. See https://bitbucket.org/eggsampler/rsc/src/bca75cc4bd27fd5d2460d5ae4ffc4484253da21f/233/src/client.java#lines-5993

File: src/Game/ReplayServer.java (hunk 1 above)
Comment (lmsv-mx123) [reply src/Game/ReplayServer.java:363]: Made into setting now defaulted to true

File: src/Game/ReplayServer.java (hunk 2 above)
Comment (lmsv-mx123) [reply src/Game/ReplayServer.java]: Placed in newer commit

File: src/Game/ReplayServer.java (hunk 3 above)
Comment (lmsv-mx123) [reply src/Game/ReplayServer.java]: Placed in newer commit

File: src/Game/ReplayServer.java (hunk 4 above)
Comment (lmsv-mx123) [reply src/Game/ReplayServer.java]: Placed in newer commit

**Pull request merged**